definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -0,0 +1,29 @@
+# This list of inputs will replace any occurrence of `{Energy Storage System Type}`
+# in a variable-definition yaml file
+
+# This list is intended for use as input energy storage system to the power sector
+
+- Electricity Storage Type:
+
+  - Energy Storage System:
+      description: Mainly to refer to the batteries
+  - Energy Storage System|Lithium-Ion:
+      description: Lithium-Ion batteries
+  - Energy Storage System|Redox Flow:
+      description: Redox Flow batteries
+  - Energy Storage System|Compressed Air:
+      description: Compressed Air batteries
+
+  - Hydro:
+      description: hydropower
+  - Hydro|Pumped Storage:
+      description: hydropower systems with short-term storage capacity
+        (including pumping turbines)
+      notes: This hydropower system has two water reservoirs at different elevations.
+        The definition follows the convention established
+        by the ENTSO-E transparency platform (https://transparency.entsoe.eu).
+  - Hydro|Reservoir:
```

Comment:
A hydro-reservoir (if not pumped) is not a storage of electricity.

Reply:
I do not totally agree with it. 
Hydro reservoirs can store water from inflows (nature way) in contrast to the pumped ones that have more or less the same behavior as batteries.
The common denominator to include `Hydro|Resevoir` in this tag is that it has many of its variables in common with respect to other storage technologies. These variables are storage (reservoir) management-related.
So, I suggest keeping it. 
Otherwise, we can generate other tags related to open-loop and close-loop storage technologies or different categories.

Reply:
This tag is used for technologies that can be used to take electricity from the grid and return it at a later point in time, so please remove all hydro technologies that are not pumped hydro and move them back to the electricity-input technologies.

If you need variables to keep track of the seasonal water storage levels, we can add new variables in a follow-up PR.

Reply:
Got it! I also agree with it.
We can merge this PR, I think.
And discuss if creating a new tag for seasonal storage levels is convenient.